app/assets/stylesheets/custom.scss

```diff
@@ -18,4 +18,9 @@
     color:deepskyblue;
     text-decoration: none;
   }
-}
\ No newline at end of file
+}
+
+.company-logo {
+  max-height: 100%;
+  max-width: 100%;
+  opacity: 1; }
```

Comment:
move "}" to next line and merge PR

Reply:
and maybe actuallt try to use bootstrap image css class "img-responsive" or something like that

Reply:
I tried using that class but it didn't work, somehow it is not accesible or something at least it doesn't show up styles on the right down corner
![image](https://user-images.githubusercontent.com/23340152/34362791-345337ca-ea7f-11e7-8e2f-18f024972ece.png)

Reply:
ok, approved